src/components/course/enrollment/tests/hooks.test.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+
+import { CourseContextProvider } from '../../CourseContextProvider';
+import { UserSubsidyContext } from '../../../enterprise-user-subsidy';
+
+import { useEnrollData, useSubsidyData } from '../hooks';
+
+const BASE_COURSE_STATE = {
+  activeCourseRun: {
+    firstEnrollablePaidSeatPrice: 7.50,
+    key: 'course_key',
+    start: '2020-02-12T10:00:00Z',
+    isEnrollable: false,
+  },
+  userSubsidyApplicableToCourse: null,
+  course: {},
+  userEnrollments: [],
+  userEntitlements: [],
+  catalog: [],
+};
+
+describe('useEnrollData tests', () => {
```

Comment:
nit: leave off `tests`. 
In general, we should be able to read the `describe` and `test` of these tests like a sentence, ie: "useEnrollData correctly extracts enroll fields". 
(Which incidentally is why I like using `it` instead of `test`; it prompts me to structure the sentence correctly)

Reply:
yes I agree